app/src/androidTest/java/com/example/admin/personallibrarycatalogue/data/TestDb.java

```diff
@@ -0,0 +1,96 @@
+package com.example.admin.personallibrarycatalogue.data;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.test.AndroidTestCase;
+import android.util.Log;
+
+import com.example.admin.personallibrarycatalogue.R;
+import com.example.admin.personallibrarycatalogue.data.DatabaseContract.BooksTable;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+
+/**
+ * Created by Admin on 26.05.2015.
+ */
+public class TestDb extends AndroidTestCase {
+
+    private static final String BOOK_TITLE = "Thinking in Java";
+    private static final String BOOK_AUTHOR = "Bruce Eckel";
+    private static final String BOOK_DESCRIPTION = "Thinking in Java is a book about the Java programming language, written by Bruce Eckel and first published in 1998";
+    public static final String LOG_TAG = TestDb.class.getSimpleName();
+
+
+    public void testCreateDb() throws Throwable {
+        mContext.deleteDatabase(LibraryDatabaseHelper.DATABASE_NAME);
+        SQLiteDatabase database = new LibraryDatabaseHelper(
+                this.mContext).getWritableDatabase();
+        assertEquals(true, database.isOpen());
+        database.close();
+    }
+
+    public void testInsert() {
+        LibraryDatabaseHelper helper = new LibraryDatabaseHelper(mContext);
```

Comment:
Just FYI. If you need to create an instance of some class in every test (like this helper) you can use ```setUp``` method to write this code one time. setUp is called every time before running your test.
See http://developer.android.com/reference/android/test/AndroidTestCase.html#setUp%28%29 (also take a look on tearDown method)